client/web/dev/utils/create-js-context.ts

```diff
@@ -26,6 +26,9 @@ export const createJsContext = ({ sourcegraphBaseUrl }: { sourcegraphBaseUrl: st
         temporarySettings: null,
         externalURL: sourcegraphBaseUrl,
         accessTokensAllow: 'all-users-create',
+        accessTokensAllowNoExpiration: false,
+        accessTokensExpirationDaysDefault: 90,
+        accessTokensExpirationDaysOptions: [90, 7, 14, 60],
```

Comment:
nit unless `90` as the first element has a special meaning... which should then accompanied with a comment.
```suggestion
        accessTokensExpirationDaysOptions: [7, 14, 60, 90],
```

Reply:
I did mention it in the site config docs.  This is admittedly kind of awkward but I wanted a way to ensure the default was actually a value that was in the list of options.  I guess I could add the default option into the list if it didn't already exist. Open to suggestions on something more obvious to users.